internal/pkg/model/manifest.go

```diff
@@ -64,6 +64,7 @@ type ConfigManifest struct {
 	RecordState `json:"-"`
 	ConfigKey
 	Paths
+	Relations Relations `json:"relations,omitempty" validate:"dive"` // relations with other objects, for example variables definition
```

Comment:
V tomto PR sa pridava moznost definovat vztahy `Relation` medzi konfiguraciami.

Reply:
Potrebujeme to na prepojenie `variables config -> config`, ... kedze vsetky ID musia byt v manifeste, pripamo v adresari projektu sa nevyskytuju.

Reply:
`Relations` by bolo mozne zapisat rovnako, ale nie je na to dôvod (Relations je `slice`, ma iba metodu `UnmarshalJSON` a tu nechceme/nepotrebujeme mat nad celou strukturou.

Dava to takto  zmysel?